client-v2/src/components/FrontsCAPIInterface/FeedItem.tsx

```diff
@@ -122,35 +125,6 @@ interface FeedItemProps {
   onAddToClipboard: (article: CapiArticle) => void;
 }
 
-// The content API drops the isLive flag on live CAPI endpoint, but keeps it in draft and print-sent.
```

Comment:
I've moved this to `utils/` as the component was large and this seemed reusable.